packages/trader/src/Modules/Trading/Components/Form/TradeParams/amount-mobile.jsx

```diff
@@ -65,40 +67,48 @@ const Basis = ({
     };
 
     return (
-        <div className='trade-params__amount-keypad'>
-            <Numpad
-                value={selected_basis}
-                format={formatAmount}
-                onSubmit={setBasisAndAmount}
-                currency={currency}
-                min={min_amount}
-                is_currency
-                render={({ value: v, className }) => {
-                    return (
-                        <div className={className}>
-                            {parseFloat(v) > 0 ? <Money currency={currency} amount={v} should_format={false} /> : v}
-                        </div>
-                    );
-                }}
-                reset_press_interval={450}
-                reset_value=''
-                pip_size={user_currency_decimal_places}
-                onValidate={validateAmount}
-                submit_label={localize('OK')}
-                onValueChange={onNumberChange}
-            />
-        </div>
+        <React.Fragment>
+            {is_turbos && (
+                <div className='trade-params__stake-container'>
+                    <TurbosInfo className='trade-container__turbos-trade-info' />
+                </div>
+            )}
+            <div className='trade-params__amount-keypad'>
+                <Numpad
+                    value={selected_basis}
+                    format={formatAmount}
+                    onSubmit={setBasisAndAmount}
+                    currency={currency}
+                    min={min_amount}
+                    is_currency
+                    render={({ value: v, className }) => {
```

Comment:
```suggestion
                    render={({ value, className }) => {
```
I think using value is more readable than v ✌🏼

Reply:
Yes, sorry, it was in original code, I'll remove it